app/version.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+function getVersionData() {
+  return new Promise((resolve, reject) => {
+    fs.readFile('version.json', 'utf8', function(err, data) {
+      if (err) {
+        reject(err);
```

Comment:
This feels suspicious...
Should be `return reject(err);` so it doesn't reject error then fall through to `try..catch` block and then resolve or re-reject?